Formula/kube-aws.rb

```diff
@@ -44,7 +44,7 @@ def install
     system "#{bin}/kube-aws"
     cluster = { "clusterName"=>"test-cluster",
                 "apiEndpoints"=>[{ "name"=>"default", "dnsName"=>"dns",
-                "loadBalancer"=>{ "createRecordSet"=>false } }],
+                                   "loadBalancer"=>{ "createRecordSet"=>false } }],
```

Comment:
Not sure what's going on here with this change. The alignment looks like it was more correct before than after.